docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -0,0 +1,369 @@
+---
+title: "Deploying a High Availability Teleport Cluster"
+description: "Deploying a High Availability Teleport Cluster"
+---
+
+When deploying Teleport in production, you should design your deployment to
+ensure that users can continue to access infrastructure during an incident in
+your Teleport cluster. In this guide, we will explain the components of a
+high-availability Teleport deployment.
+
+## Overview
+
+A high-availability Teleport cluster revolves around a group of redundant
+`teleport` processes, each of which runs the Auth Service and Proxy Service,
+plus the infrastructure required to support them. 
+
+This includes:
+
+- **A Layer Four load balancer** to direct traffic from users and services to an
+  available `teleport` process.
+- A **key-value store** for Auth Service state and audit events that all
+  `teleport` instances can access. This requires permissions for `teleport`
```

Comment:
I would call them `auth services` or auth instances` to be clear it is only the auth service process that needs to reach the key-value store

Same for the object storage service and license file sections

Reply:
👍 I've mentioned the specific services throughout the guide